docs/rest-api/api/driveitem_createuploadsession.md

```diff
@@ -37,10 +37,12 @@ Once the last byte of the file has been uploaded the upload session is completed
 
 ```http
 POST /drives/{driveId}/items/{itemId}/createUploadSession
+POST /drives/{drive}/items/{itemId}:/{fileName}:/createUploadSession
```

Comment:
Please change {drive} to be {driveId} for consistency with the other examples.

Reply:
@chackman Will do!

Reply:
Done